docs/prefix.md

```diff
@@ -0,0 +1,80 @@
+(named-prefixes)=
+# Named IP Prefixes
+
+In most lab topologies, you probably don't care about the exact IP addresses and subnets. Defining IPv4 and IPv6 prefixes in [addressing pools](address pools) is good enough.
+
+If you want to tighten control over IP address allocation, use the **prefix** attribute on links or VLANS or the **ipv4**/**ipv6** attributes on node interfaces.
+
+However, there are scenarios in which you have to use the same prefix in multiple places, for example:
+
+* Using a link prefix in the lab validation code
+* Using a link prefix in a [prefix list](module-routing)
+* Using the same prefix on multiple links, for example, to implement a stretched subnet using a technology not supported by _netlab_.
+
+You can use *named IP prefixes* in all three scenarios. The named prefixes are defined in the top-level **prefix** dictionary. The dictionary keys are prefix names, the values are dictionaries defining individual prefixes. The prefix values can include **ipv4**, **ipv6**, **pool** and **[allocation](addr-allocation-sequential)** attributes.
```

Comment:
I don't know if it's me (maybe yes), but cannot find the `allocation` specific documentation.